parlai/core/torch_agent.py

```diff
@@ -775,7 +775,7 @@ def _init_cuda_buffer(self, model, criterion, batchsize, maxlen):
             try:
                 print('preinitializing pytorch cuda buffer')
                 dummy = torch.ones(batchsize, maxlen).long().cuda()
-                sc = model(dummy, dummy)[1]
+                sc = model(dummy, dummy)[0]
```

Comment:
this needs an explanation. It's assuming model has a particular interface and that interface isn't clear to me right now.

Reply:
(Assuming this is related to the hidden, encoder_out, attn_mask from earlier)

Reply:
agreed, I need to find a better way to extract the scores here or move it back to s2s